lib/common/attrd_client.c

```diff
@@ -278,14 +278,7 @@ pcmk__node_attr_request_clear(crm_ipc_t *ipc, const char *host,
 const char *
 pcmk__node_attr_target(const char *name)
 {
-    if (pcmk__strcase_any_of(name, "auto", "localhost", NULL)) {
-        name = NULL;
-    }
-
-    if(name != NULL) {
-        return name;
-
-    } else {
+    if (name != NULL && pcmk__strcase_any_of(name, "auto", "localhost", NULL)) {
```

Comment:
name == NULL ||

or

    if ((name != NULL) && !pcmk__strcase_any_of(...)) {
        return NULL;
    }

Reply:
I think I meant to do the latter, but I like the former more.  I must have looked at this a half-dozen times, but I think I was a little distracted the whole time anyway.